test/mitmproxy/addons/test_proxyauth.py

```diff
@@ -109,6 +121,24 @@ def test_check():
         )
         assert not up.check(f)
 
+        ctx.configure(
```

Comment:
Am I right in the assumption that this test connects to the `ldap.forumsys.com` server? I see why you do this here, but we'd be really keen to keep our tests running even if there is no network connection. How difficult would it be to run a minimal ldap server for the tests?